packages/gatsby-theme/src/components/Documentation/Layout/index.tsx

```diff
@@ -37,6 +37,11 @@ const Layout: React.FC<PropsWithChildren<{ currentPath: string }>> = ({
       <button
         className={cn(styles.sideToggle, isMenuOpen && styles.opened)}
         onClick={toggleMenu}
+        aria-label={
+          isMenuOpen
+            ? 'Close documentation sidebar'
+            : 'Open documentation sidebar'
+        }
```

Comment:
Tweaked to add a target for tests. Changes when open/closed in an attempt to be able to target something visible to verify the sidebar is open when trying to fix the odd sidebar-not-opening issue, but it didn't work. Still kept since I think it's good for a11y.